README.md

```diff
@@ -50,9 +50,11 @@ Project management [[See on Waffle.io]](https://waffle.io/decidim/decidim)
 
 TLDR: install gem, generate a Ruby on Rails app, enjoy.
 
-```
-gem install decidim
-decidim decidim_application
+```bash
+$ gem install decidim
```

Comment:
I'd expect explanation to be on top of the command, not below

Reply:
It turns out that the linter forces me to actually show the output of the command *after* we use a `$` in a code block. This is the only acceptable solution I found xD. Applies to the rest of the comments.

Reply:
What the hell, totally unexpected. Can't this be configured? I feel it's too weird 😕

Reply:
Well, it makes total sense - when you're showing an example of a command being executed, you should show its output. It was really impractical to me (in this PR) to actually go through all of them, but I think it's reasonable for the new markdown files.

Reply:
Meh, not sure, but won't block the PR.